include/eve/memory/aligned_ptr.hpp

```diff
@@ -450,7 +450,7 @@ namespace eve
   template <typename T>
   EVE_FORCEINLINE auto previous_aligned_address(T* p) noexcept
   {
-    return previous_aligned_address(p, eve::expected_cardinal<std::remove_cvref_t<T>>{});
+    return previous_aligned_address(p, eve::expected<std::remove_cvref_t<T>>);
```

Comment:
I think eve::exprected can also be removed, and eve_expected_cardinal_t used instead